### ISSUE TITLE
List items with `#` after `-` are rendered as titles in the "+" editor themes

### ISSUE THREAD

**Issue by bomberstudios** [MacDownApp/macdown#107]
Example:

![screenshot 2014-08-05 23 45 36](https://cloud.githubusercontent.com/assets/3832/3819417/736e703e-1cea-11e4-8702-cd954c7441a4.png)

**Comment by jordanst3wart** [comment]
Is this issue related?
![09_feb_sceenshot_macdown](https://cloud.githubusercontent.com/assets/12145372/12902278/50d71c14-cf14-11e5-91f1-4bb40f2bd81a.jpg)

**Comment by bomberstudios** [comment]
No, that is expected behaviour. That lone `-` on the last line is telling Markdown that the line before is an `h2` (see https://daringfireball.net/projects/markdown/syntax#header for an explanation)

**Comment by jordanst3wart** [comment]
Okay thanks.
